app/gradle.properties

```diff
@@ -1,3 +1,3 @@
 kotlin.code.style=official
-release_version=0.9.0
+release_version=0.9.1
```

Comment:
note: We didn't add any new functionality - why do we increment the version?

Reply:
We solved the `pull continuation failure when column with oracle DATE type is used for update query.` problem